util/build_configs/cray-internal/setenv-xc-aarch64.bash

```diff
@@ -141,8 +141,10 @@ if [ -z "$BUILD_CONFIGS_CALLBACK" ]; then
             --launcher=$launchers \
             --substrate=$substrates \
             --locale-model=$locale_models \
-            --auxfs=$auxfs
+            --auxfs=$auxfs \
+            notcompiler
```

Comment:
I _think_ we want this to be `-- notcompiler` like above for `-- compiler`

Reply:
OK, I'll do that